pkg/api/validation/validation_test.go

```diff
@@ -118,6 +136,7 @@ func TestValidateAnnotations(t *testing.T) {
 		{"1234/5678": "bar"},
 		{"1.2.3.4/5678": "bar"},
 		{"UpperCase123": "bar"},
+		{"a": strings.Repeat("b", 64*(1<<10)-1)},
```

Comment:
honestly this is a complicated way to spell "65535" or "(64*1024)-1" :) I appreciate the attempt at self-describing, but I think bit shifts are harder for most people to follow, especially when teh expanded value is in the "memorizable range"